server/src/main/java/org/opensearch/common/cache/refcounted/stats/CacheStats.java

```diff
@@ -0,0 +1,277 @@
+     * value can result in multiple misses, all returning the results of a single cache load
+     * operation.
+     *
+     * @return the number of times {@link RefCountedCache} lookup methods have returned an uncached (newly
+     * loaded) value, or null
+     */
+    public long missCount() {
+        return missCount;
+    }
+
+    /**
+     * Returns the ratio of cache requests which were misses. This is defined as
+     * {@code missCount / requestCount}, or {@code 0.0} when {@code requestCount == 0}.
+     * Note that {@code hitRate + missRate =~ 1.0}. Cache misses include all requests which
+     * weren't cache hits, including requests which resulted in either successful or failed loading
+     * attempts, and requests which waited for other threads to finish loading. It is thus the case
+     * that {@code missCount &gt;= loadSuccessCount + loadFailureCount}. Multiple
+     * concurrent misses for the same key will result in a single load operation.
+     *
+     * @return the ratio of cache requests which were misses
+     */
+    public double missRate() {
+        long requestCount = requestCount();
+        return (requestCount == 0) ? 0.0 : (double) missCount / requestCount;
+    }
+
+    /**
+     * Returns the number of times an entry has been removed explicitly.
+     *
+     * @return the number of times an entry has been removed
+     */
+    public long removeCount() {
+        return removeCount;
+    }
+
+    /**
+     * Returns the sum of weights of explicitly removed entries.
+     *
+     * @return the sum of weights of explicitly removed entries
+     */
+    public long removeWeight() {
+        return removeWeight;
+    }
+
+    /**
+     * Returns the number of times an entry has been replaced.
+     *
+     * @return the number of times an entry has been replaced
+     */
+    public long replaceCount() {
+        return replaceCount;
+    }
+
+    /**
+     * Returns the number of times an entry has been evicted. This count does not include manual
+     * {@linkplain RefCountedCache#remove removals}.
+     *
+     * @return the number of times an entry has been evicted
+     */
+    public long evictionCount() {
+        return evictionCount;
+    }
+
+    /**
+     * Returns the sum of weights of evicted entries. This total does not include manual
+     * {@linkplain RefCountedCache#remove removals}.
+     *
+     * @return the sum of weights of evicted entities
+     */
+    public long evictionWeight() {
+        return evictionWeight;
+    }
+
+    /**
+     * Returns a new {@code CacheStats} representing the difference between this {@code CacheStats}
+     * and {@code other}. Negative values, which aren't supported by {@code CacheStats} will be
+     * rounded up to zero.
+     *
+     * @param other the statistics to subtract with
+     * @return the difference between this instance and {@code other}
+     */
+
+    public CacheStats minus(CacheStats other) {
```

Comment:
Are the `plus()` and `minus()` methods used anywhere?

Reply:
nope they are not used anywhere for now but there are few more methods which are not used but will be used once cache stats is exposed to nodes stats and maybe `plus` will be one of them. For now will remove `plus` and `minus` and if needed will add them back

Reply:
Please remove the unused code, methods will be introduced when needed